x/pairing/types/epoch_cu.go

```diff
@@ -0,0 +1,65 @@
+package types
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	UniqueEpochSessionPrefix      = "UniqueEpochSession/"
+	ProviderEpochCuPrefix         = "ProviderEpochCu/"
+	ProviderConsumerEpochCuPrefix = "ProviderConsumerEpochCu/"
+)
+
+func UniqueEpochSessionKey(provider string, project string, chainID string, sessionID uint64) []byte {
```

Comment:
can we make the key by the order of:
provider chainid consumer sessionid

Reply:
done. see https://github.com/lavanet/lava/commit/f25f922722ccf829a32f92024743b5e9608495e8